### ISSUE TITLE
An error was found that was difficult to resolve after manually reroot

### ISSUE THREAD

**Issue by Five296** [geronimp/graftM#283]
graftM create --sequences bglX_all_rename_dup.fasta  --taxtastic_taxonomy graftm_create_taxonomy.bglX_all_rename_dup.csv --taxtastic_seqinfo graftm_create_seqinfo.bglX_all_rename_dup.csv --alignment graftm_create_alignment.bglX_all_rename_dup.faa  --rerooted_tree 1.tree

                             GraftM 0.15.0
01/17/2024 11:57:12 PM WARNING: Program found with multiple commands: taxit FastTreeMP hmmalign mafft. Arbitrarily selecting FastTreeMP

                            CREATE

                   Joel Boyd, Ben Woodcroft

                                                    /
              >a                                   /
              -------------                       /
              >b                        |        |
              --------          >>>     |  GPKG  |
              >c                        |________|
              ----------

01/17/2024 11:57:12 PM INFO: Building gpkg for bglX_all_rename_dup.gpkg
01/17/2024 11:57:12 PM INFO: Reading taxonomy from taxtastic taxonomy and seqinfo files
01/17/2024 11:57:12 PM INFO: Checking for duplicate sequence names
01/17/2024 11:57:12 PM INFO: Building HMM from alignment
01/17/2024 11:57:13 PM INFO: Filtered 0 short sequences from the alignment
01/17/2024 11:57:13 PM INFO: 320 sequences remaining
01/17/2024 11:57:13 PM INFO: Checking for incorrect or fragmented reads
01/17/2024 11:57:59 PM INFO: Building HMM from alignment
01/17/2024 11:58:00 PM INFO: Filtered 0 short sequences from the alignment
01/17/2024 11:58:00 PM INFO: 320 sequences remaining
01/17/2024 11:58:00 PM INFO: Deduplicating sequences
01/17/2024 11:58:00 PM INFO: Removed 0 sequences as duplicates, leaving 320 non-identical sequences
01/17/2024 11:58:00 PM INFO: Removing duplicates from tree
01/17/2024 11:58:00 PM INFO: Generating log file
01/17/2024 11:58:14 PM INFO: Creating reference package
01/17/2024 11:58:15 PM INFO: Creating diamond database
01/17/2024 11:58:15 PM INFO: Compiling gpkg
01/17/2024 11:58:15 PM INFO: Cleaning up
01/17/2024 11:58:15 PM INFO: Testing gpkg package works
Traceback (most recent call last):
  File "/data/home/wuzy/miniconda3/envs/mNGS/bin/graftM", line 420, in <module>
    Run(args).main()
  File "/data/home/wuzy/miniconda3/envs/mNGS/lib/python3.11/site-packages/graftm/run.py", line 672, in main
    self.create.main(
  File "/data/home/wuzy/miniconda3/envs/mNGS/lib/python3.11/site-packages/graftm/create.py", line 889, in main
    self._test_package(output_gpkg_path)
  File "/data/home/wuzy/miniconda3/envs/mNGS/lib/python3.11/site-packages/graftm/create.py", line 546, in _test_package
    extern.run(cmd)
  File "/data/home/wuzy/miniconda3/envs/mNGS/lib/python3.11/site-packages/extern/__init__.py", line 41, in run
    raise ExternCalledProcessError(process, command)
extern.ExternCalledProcessError: Command graftM graft --forward /tmp/tmpq6zrcmo4.fa --graftm_package bglX_all_rename_dup.gpkg --output_directory /tmp/tmpppt30s5u --force returned non-zero exit status 1.
STDERR was: b'01/17/2024 11:58:15 PM INFO: Working on tmpq6zrcmo4\n01/17/2024 11:58:17 PM INFO: 10 read(s) detected\n01/17/2024 11:58:17 PM INFO: aligning reads to reference package database\n01/17/2024 11:58:20 PM INFO: Filtered 0 short sequences from the alignment\n01/17/2024 11:58:20 PM INFO: 10 sequences remaining\n01/17/2024 11:58:20 PM INFO: Placing reads into phylogenetic tree\nTraceback (most recent call last):\n  File "/data/home/wuzy/miniconda3/envs/mNGS/bin/graftM", line 420, in <module>\n    Run(args).main()\n  File "/data/home/wuzy/miniconda3/envs/mNGS/lib/python3.11/site-packages/graftm/run.py", line 620, in main\n    self.graft()\n  File "/data/home/wuzy/miniconda3/envs/mNGS/lib/python3.11/site-packages/graftm/run.py", line 509, in graft\n    taxonomic_assignment_time, assignments=self.p.place(REVERSE_PIPE,\n                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/data/home/wuzy/miniconda3/envs/mNGS/lib/python3.11/site-packages/graftm/timeit.py", line 10, in timed\n    result = method(*args, **kw)\n             ^^^^^^^^^^^^^^^^^^^\n  File "/data/home/wuzy/miniconda3/envs/mNGS/lib/python3.11/site-packages/graftm/pplacer.py", line 189, in place\n    jplace = self.pplacer(files.jplace_output_path(), args.output_directory, files.comb_aln_fa(), args.threads)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/data/home/wuzy/miniconda3/envs/mNGS/lib/python3.11/site-packages/graftm/pplacer.py", line 28, in pplacer\n    extern.run(cmd)\n  File "/data/home/wuzy/miniconda3/envs/mNGS/lib/python3.11/site-packages/extern/__init__.py", line 41, in run\n    raise ExternCalledProcessError(process, command)\nextern.ExternCalledProcessError: Command pplacer -j 5 --verbosity 0 --out-dir /tmp/tmpppt30s5u -c bglX_all_rename_dup.gpkg/bglX_all_rename_dup.gpkg.refpkg /tmp/tmpppt30s5u/combined_alignment.aln.fa returned non-zero exit status -6.\nSTDERR was: b"pplacer: loadlocale.c:129: _nl_intern_locale_data: Assertion `cnt < (sizeof (_nl_value_type_LC_TIME) / sizeof (_nl_value_type_LC_TIME[0]))\' failed.\\n"STDOUT was: b\'\'\n'STDOUT was: b'\n                             GraftM 0.15.0\n\n                                GRAFT\n\n                       Joel Boyd, Ben Woodcroft\n\n                                                         __/__\n                                                  ______|\n          _- - _                         ________|      |_____/\n           - -            -             |        |____/_\n           - _     >>>>  -   >>>>   ____|\n          - _-  -         -             |      ______\n             - _                        |_____|\n           -                                  |______\n            \n'

**Comment by Five296** [comment]
[bglX.zip](https://github.com/geronimp/graftM/files/13974357/bglX.zip)
Here's my file

**Comment by wwood** [comment]
Hi,

This appaears to be a problem with your pplacer installation, not graftM directly.

**Comment by Five296** [comment]
Thank you for your response. I used 'mamba create -n graftM graftM' to download graftM and its dependencies, such as pplacer (latest version v1.1.alpha19-0-g807f6f3). I then extracted 20 sequences from the dataset successfully creating a gpkg subset, but the original dataset consistently failed. After trying various methods, I discovered that using '--rerooted' caused such issues, while switching to '--no-tree' resolved the problem. Does '--no-tree' compromise the reliability of the results?
